content/glossary.md

```diff
@@ -34,88 +35,82 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
 
 ---
 
-**Sampling**  
-Is the mechanism that determines how often the system will trace a request.  
-There are  3 types of sampling:  
-
-1. **Always sample:**  
-  
-    - With this sampling, every single request is traced.  
-		
-2. **Probabilistic sampling:**  
-	
-	- Here a probability is set e.g specify 1 in 10000 or 0.0001 and the system will sample only 1 in 10000 requests. This entirely depends on your QPS.  
-		
-3. **Never sample:**  
-	
-	- Here you’ve asked that the system never traces any request.  
+**Sampling**
 
----
+Sampling determines how often requests will be traced.
+There are three types of sampling made available from OpenCensus libraries:  
 
-**Measure**  
-Any quantifiable measurement/metric that you would like to track, it could be number of requests, number of failed responses, bytes downloaded, cache misses, number of radios purchsed etc.  
+* **Always sample:** With this sampling, every single request is traced.  
+* **Probabilisticly sample:** A probability is set (e.g 0.0001) and the libraries will sample only 1 in 10000 requests.
+* **Never sample:** No request is traced.  
 
 ---
 
-**Aggregation**  
-The style of characterizing measures. We have 4 types of Aggregations:  
-
-1. **Count Aggregation:**  
+**Measure**  
+Any quantifiable metric. Examples of measures are number of requests, number of failed responses, bytes downloaded, cache misses, number of transactions, etc.  
 
-    - Characterizing measures as a sum of increasing quantiles e.g number of requests overtime always increases and is quantified by a natural number.  
-	i.e. >= 1  
-  
-2. **Distribution Aggregation:**  
+---
 
-    - Characterizing measures as belonging to buckets with ranges on sharp boundaries e.g. <10, <100, <1000 which distinctly identify values of 112, 99, 17 as belonging to buckets <1000, <100 and <10 respectively.  
-  
-3. **Mean Aggregation:**  
+**Aggregation**
 
-    - Characterizing measures over the mean of all of the values and maintains the mean value.  
-  
-4. **Sum Aggregation:**  
+OpenCensus doesn't export each collected measurement. It preaggregates measurements in the process.
+OpenCensus provides the following aggregations:  
 
-    - Characterizing measures over the total of values and maintains the value as a sum.  
+* **Count:** Reports the count of the recorded data points. For example, number of requests.
+* **Distribution:** Reports the histogram distribution of the recorded measurements.
+* **Mean:** Reports the mean of the recorded measurements.
+* **Sum:** Reports the sum of the recorded measurements.
+* **Last Value:** Reports on the last recorded measurement and drops everything else.
 
 ---
 
-**Tags**  
-Tags are key-value pairs that are used to store information about metrics for example;  
+**Tags**
 
-+ ip=10.32.103.12  
+Tags are key-value pairs that can be recorded with measurements.
+They later used to breakdown the collected data in the metric collection backend.
```

Comment:
s/They later used to/They are later used to

Reply:
Done.